### DISCUSSION TITLE
Unable to reset admin password

### DISCUSSION THREAD

**Discussion by abhilash-cuberoot** [pocketbase/pocketbase#5064] [Q&A] [answered]
`./pocketbase admin update 'admin@example.com' 'XjBeyHfyr2wc4W*C8SUzk9qK7K#ejfo'`
which results ./pocketbase admin update 'admin@example.com' 'XjBeyHfyr2wc4W*C8SUzk9qK7K#ejfo'
However logging with admin@example.com and password XjBeyHfyr2wc4W*C8SUzk9qK7K#ejfo doesn't work.

I have used simple passwords with and without quotes, nothing is working.
The SMTP is not set correctly so I am unable to get email.
is there any way to resolve this?

**Comment by ganigeorgiev** [comment]
Testing locally with the latest PocketBase v0.22.13 binary works correctly for me.

Do you see a success message "Successfully changed admin ..." after running the above command?

In any case, another alternative could be to manually update the password hash in the database as shown in https://github.com/pocketbase/pocketbase/discussions/2496#discussioncomment-5926734.

**Comment by abhilash-cuberoot** [comment]
Yes I see "Successfuly changed admin ..." 
Getting this message "Failed to authenticate" when applying to admin panel.

I have tried update the db directly as well like #2496  still not working.
I can see that the db has changed by `sqlite3 data.db "SELECT * from _admins"`

**Reply by ganigeorgiev** [reply]
If you see the success message then the password should have been updated. If even after manually changing the db entry it still doesn't work then there is definitely something else going on in your environment but I don't think I can help you further.

Make sure that you are running the command against the correct env/db file and that you are no typos/spaces when submitting the email/password in the Admin UI.

For more detailed error messages you can also try to start the executable with `--dev`.

**Comment by abhilash-cuberoot** [comment, marked as answer]
Thanks, I have restarted the pocketbase and it worked. It seems to me that the token needs to be updated which did when I restarted pocketbase.

**Reply by ganigeorgiev** [reply]
Tokens are not stored in the database so there is nothing to update and the app restart shouldn't have effect on the password change. Again, something else is happening in your env (ex. testing against a different `pb_data`) but without clear reproducible steps there is no much I can help with.

**Reply by abhilash-cuberoot** [reply]
I did see token in database using `sqlite3 data.db "SELECT * from _admins"`

**Reply by ganigeorgiev** [reply]
If you mean the `passwordHash` field it is not a token, it is the bcrypt hash of the plain password string. The `tokenKey` field is also not an auth token (it is the unique user token secret).

In any case I'll stop engaging further to avoid further confusion. If you are still not able to resolve it, please provide clear reproducible steps or a **minimal** reproducible repo.
